source/extensions/common/proxy_protocol/proxy_protocol_header.h

```diff
@@ -61,6 +65,9 @@ void generateProxyProtoHeader(const envoy::config::core::v3::ProxyProtocolConfig
 // Generates the v2 PROXY protocol local command header and adds it to the specified buffer
 void generateV2LocalHeader(Buffer::Instance& out);
 
+// Generates the v2 PROXY protocol header including the TLV vector into the specified buffer.
+void generateV2Header(const Network::ProxyProtocolData& prox_proto_data, Buffer::Instance& out);
```

Comment:
s/prox_proto_data/proxy_proto_data/

Reply:
Also should this go up by the other three generateV2Header function declarations?

Reply:
Done!